### PR TITLE
SWI-1687: [.NET] Unit Tests for API Models

### PR DESCRIPTION
**Pull request by juliajanu**

### CODE REVIEW

File: config.yml
Comment (ajrice6713) [inline config.yml:1]: can we rename this to `openapi-config.yml`? Just to be consistent + explicit

**Review by ajrice6713 — CHANGES_REQUESTED** [review]
other than the one knitpicky thing, this looks great @juliajanu!

File: config.yml
Comment (juliajanu) [reply config.yml:1]: Is there a difference between the config.yml and the openapi-config.yml because that already exists...

Comment (ckoegel) [reply config.yml:1]: there should only be one, I'd update `openapi-config.yml` to include the stuff you need from `config.yml` and then get rid of `config.yml`

File: openapi-config.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
----
-apiName: Bandwidth
+TargetFramework: netstandard2.1
```
Comment (ckoegel) [inline openapi-config.yml:1]:
```suggestion
targetFramework: netstandard2.1
```

**Review by ajrice6713 — APPROVED** [review]

**Pull request merged**